packages/react/src/Link/Link.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2021 Gemeente Utrecht
+ * Copyright (c) 2021 Robbert Broersma
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Link as CommunityLink } from '@utrecht/component-library-react'
+import clsx from 'clsx'
+import { AnchorHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+
+export type LinkColor = 'default' | 'black' | 'white'
+
+export type LinkVariant = 'default' | 'bold'
+
+export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string
+  placeholder?: string
```

Comment:
Not the same as a `placeholder` as used in inputs? I would opt for a different prop name. I guess in general I prefer different names if they are the same as the default HTML attributes. Same for `color`

Reply:
From the Utrecht Link
```
export interface LinkProps extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'placeholder'> {
  boxContent?: boolean;
  external?: boolean;
  placeholder?: boolean;
}